src/rbus/rbus.c

```diff
@@ -448,6 +448,8 @@ rbusError_t rbusProperty_initFromMessage(rbusProperty_t* property, rbusMessage m
     err= rbusValue_initFromMessage(&value, msg);
     rbusProperty_SetValue(*property, value);
     rbusValue_Release(value);
+    if(name)
```

Comment:
There is no allocation of memory for name..  it just a pointer pointin to memory location of rbusMessage.. as long as rbusMessage is released, it should be fine..